config/locales/doc_auth/es.yml

```diff
@@ -214,7 +215,9 @@ es:
       text_message: Enviamos un mensaje a su teléfono
       upload_from_computer: Continuar en esta computadora
       upload_from_phone: Utilice su teléfono para tomar las fotos
+      verify_at_post_office: Verifique su identidad en una Oficina de Correos
```

Comment:
Oh gosh I feel bad only finding ridiculously trivial things (I guess that's because this looks great?), but it looks like @gina-yamada's PR moves "Oficina de Correos" to lowercase. Was that part of an official change we want to duplicate here?

(Also, I'm not actually trying to review copy changes in other languages at all; my eyes just happened to notice this one for some reason.)

Reply:
Good catch! Yes, Carmen and I just discussed this in my [PR](https://github.com/18F/identity-idp/pull/9494)  because I also had the C capitalized. If you do a global search for Post Office in Spanish, they are all lower case.

Reply:
@allthesignals @gina-yamada Hey! Two days ago we were debating whether we should keep the Spanish and French version of Post Office in upper case, and we looked more into it and on the USPS website, the translation to Spanish shows it with lowercase. So we're going to follow that approach for French and Spanish of keeping the translation of Post Office lower case.

Reply:
Yes! Reviewing it now. I will comment when I am finished.